### ISSUE TITLE
Whatsapp notification mute action is misbehaving 

### ISSUE THREAD

**Issue by ramysami** [8bitPit/Niagara-Issues#2230]
When recieving multiple Whatsapp notifications , the mute button is not corresponding to that specific message, instead it mutes a random one

**Comment by Amorphous404** [comment]
Hi @ramysami, Thank you for reporting; Niagara Launcher replicates the app notifications/action buttons from the Android notification drawer. Do you face the same when you perform the mute action via the Notification drawer?

If not, could you please provide this additional information;

1. Device name
2. Android version
3. WhatsApp version
4. Niagara version

**Comment by ramysami** [comment]
Yeah I can confirm when I tap mute on a specific chat from the normal notifications it works properly, but when doing so from Niagra the mute buttons always behave as it is the first notification regardless. 



Product name : Galaxy A51
MODEL name : SM-A515F/DSN
Android 13
Whatsapp version: 2.23.6.76
Niagara: 1.8.15

**Comment by Amorphous404** [comment]
Thank you for checking and providing the details. We'll update you once we know more about the issue.
